dnplab/dnpIO/topspin.py

```diff
@@ -524,25 +516,28 @@ def topspin_ser_phase_cycle(path, expNum, paramFilename="acqus"):
     return output
 
 
+"""
+## NOT NEEDED ##
 def import_topspin_dir(path):
-    """
+
     Import directory of Topspin data and return as dictionary
 
     Args:
         path (str): Directory of data
 
     Returns:
         dict: Topspin data. Keys correspond to folder name. Values correspond to dnpdata with topspin data for each folder.
-    """
+
 
     dirFiles = [x for x in _os.listdir(path) if _os.path.isdir(_os.path.join(path, x))]
 
     dataDict = {}
     for expNum in dirFiles:
         try:
-            tempData = import_topspin(path, expNum)
+            tempData = import_topspin(path)
             dataDict[expNum] = tempData
         except:
             pass
 
     return dataDict
+"""
```

Comment:
Delete the lines instead of comment it out.

Reply:
done